docs/products/access/appendix/templates/templates.md

```diff
@@ -214,4 +214,66 @@ The name of the template file must be `endsession-reject.html`.
+
+### Available Messages
+
+| Key                             | Description                                                 |
+|---------------------------------|-------------------------------------------------------------|
+| pages.userCode.title            | Title for the user code page.                               |
+| userCode.header.enter           | Header text when the user is entering the code.             |
+| userCode.header.confirm         | Header text when the user is confirming the code.           |
+| userCode.intro.enter            | Introductory text explaining the code entry process.        |
+| userCode.intro.confirm          | Introductory text explaining the confirmation process.      |
+| userCode.submit                 | Label used for the submit button.                           |
+| userCode.error.invalidCode      | Error message displayed when an incorrect code is entered.  |
+| userCode.error.invalidCsrfToken | Error message displayed when there's a CSRF token mismatch. |
+
+### Available Variables
+
+| Variable Name    | Description                                                 |
+|------------------|-------------------------------------------------------------|
+| userCode         | The actual user code that needs to be confirmed or entered. |
+| invalidUserCode  | Boolean indicating if the entered user code is invalid.     |
+| invalidCsrfToken | Boolean indicating if there's a CSRF token mismatch.        |
+
+
+## Device Authorization Status Template
+
+### Introduction
+
+The Device Authorization Status template displays the status of a device's authorization process. It is used to communicate with the end
+user whether the device connection was successful, expired, already connected, or invalid.
+
+### Naming Convention
+
+The name of the template file must be `device-authorization-status.html`.
+
+### Available Messages
+
+| Key                                             | Description                                            |
+|-------------------------------------------------|--------------------------------------------------------|
+| pages.deviceAuthStatus.title.success            | Title when the device is successfully connected.       |
+| pages.deviceAuthStatus.title.expired            | Title when the user code has expired.                  |
+| pages.deviceAuthStatus.title.already_authorized | Title when the device is already connected.            |
+| pages.deviceAuthStatus.title.invalid_user_code  | Title when an invalid code is entered.                 |
+| deviceAuthStatus.content.success                | Content message for successful connection.             |
+| deviceAuthStatus.content.expired                | Content message when the code has expired.             |
+| deviceAuthStatus.content.already_authorized     | Content message when the device is already authorized. |
+| deviceAuthStatus.content.invalid_user_code      | Content message when an invalid code is entered.       |
```

Comment:
The message keys are following two casing strategies (camel and underscore), please inline.